### ISSUE TITLE
How to enable/update Analyzers in training VM ?

### ISSUE THREAD

**Issue by vletoux** [TheHive-Project/CortexDocs#63]
Seems stupid but in https://github.com/TheHive-Project/TheHiveDocs/blob/master/training-material.md, it is indicated that "With the new version, analyzers are disabled by default. The training VM is delivered with Abuse Finder, File_Info, Msg_Parser and MaxMind GeoIP enabled."

Ok, so I have to update them (https://github.com/TheHive-Project/CortexDocs/blob/master/installation/install-guide.md#updating)

But I have to go to a specific directory and update the git code:
/opt/Cortex-Analyzers

But this directory doesn't exist in the VM
![image](https://user-images.githubusercontent.com/10632326/78266487-24732180-7506-11ea-94ac-0bc3c6dbcc6a.png)

Ok, maybe the git directory is located elsewhere, let's locate it:
![image](https://user-images.githubusercontent.com/10632326/78266567-41a7f000-7506-11ea-9d2c-39651f80db82.png)

Maybe there is an alternative directory configured in cortex:
![image](https://user-images.githubusercontent.com/10632326/78266675-62704580-7506-11ea-9919-6ff02226887e.png)

In short I've used the training VM to play with the application, and I see only a few analyzers.

I suggest to modify the VM or to update the documentation to have the same path in the VM that in the doc.

**Comment by vletoux** [comment]
seems that the file /etc/cortex/application.conf needs to be edited to comment the urls and uncomment the path
![image](https://user-images.githubusercontent.com/10632326/78269173-82553880-7509-11ea-8d79-5c4dd1c9fe05.png)

Then issue a git clone and then follow the instruction to update the python requirements
